### DISCUSSION TITLE
Fixed Cost behaves oddly

### DISCUSSION THREAD

**Discussion by GuyBenhaim** [google/or-tools#2957] [Routing (and legacy CP) questions]
Seems that vehicle FixedCost is evaluated vs. a single Node's disjunction penalty: if the FixedCost is slightly higher the vehicle is not used and similarly **no vehicle was used and all Nodes were dropped**.
10 Vehicles, 80 Orders.

If Solver would have allocated the vehicle, it would server multiple Nodes and reduce penalty substantially.

When FixedCost was even a bit lower than disjunction penalty (of one Node) all necessary vehicles were allocated and served the Nodes.

Looks like the **criteria to allocate was just based one vehicle vs. one Node**. Correct?
If I'm not wrong, this creates undesired results.

**Comment by Mizux** [comment]
seems related to #880

**Comment by GuyBenhaim** [comment]
Indeed, I thought 880 was closed.
Any insights?

**Comment by GuyBenhaim** [comment]
Hi,
Is my observation correct regarding how FixedCost affects search?

**Comment by Mizux** [comment]
Yes, according to the team it is an open issue.
Don't have ETA for solving this one, sorry

**Comment by GuyBenhaim** [comment]
So, the behavior I described should be expected?

**Comment by Mizux** [comment]
AFAIK during the local search there is no heuristic which try to insert a "bunch of nodes" **then** see if it reduce the objective value or try without taking into account the fixed cost first then compare both result...

I'll try to write a minimal example to show the issue

**Comment by gabisurita** [comment]
Hi, I've seem this has been removed from the backlog. Was there any progress on this issue?

**Comment by Mizux** [comment]
sorry, team is still aware of it but AFAIK currently there is no one working on it.
